### PR TITLE
[[ Bugfix 11323 ]] Add new set ops and into form

### PR DESCRIPTION
**Pull request by runrevmark**
This patch adds difference and symmetric difference set operation commands, and adds an 'into' clause to all four.

### CODE REVIEW

File: tests/lcs/core/array/setoperations.livecodescript [hunk 1]
```diff
@@ -109,6 +126,118 @@ on TestNestedArrayRecursiveUnion
...
+
+/*
+Semantics of array symmetric difference:
+
+function ArraySymmetricDifference(pLeft, pRight)
+	repeat for each key tKey in pRight
+		if tKey is not among the keys of pLeft then
+			put pRight[tKey] into pLeft[tKey]
+		else 
+			delete pLeft[tKey]
```
Comment (livecodeali) [inline tests/lcs/core/array/setoperations.livecodescript]: not a huge deal but `delete variable pLeft[tKey]`

File: tests/lcs/core/array/setoperations.livecodescript [hunk 2]
```diff
@@ -209,3 +356,114 @@ on TestNestedArrayRecursiveIntersect
...
+			tOut[1][2] is "c"
+end TestArrayRecursiveIntersectInto
+
+/*
+Semantics of array difference:
+
+function ArrayDifference(pLeft, pRight)
+	repeat for each key tKey in pLeft
+		if tKey is among the keys of pRight then
+			delete pLeft[tKey]
```
Comment (livecodeali) [inline tests/lcs/core/array/setoperations.livecodescript]: missing `variable` here too

**Comment by runrevmark** [comment]
@livecodeali - I've tweaked the things you've found - so should be good for review!

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 66bb4ac

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 66bb4acdecb4f8414869ceb6f0994025b4ca930c

**Comment by livecode-vulcan** [comment]
😎 test success 66bb4acdecb4f8414869ceb6f0994025b4ca930c

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/308)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/314)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/294)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/237)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/237)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/303)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/308)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/306)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/312)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/306)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/312)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/331)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/302)

**Pull request merged**